arcgis-ios-sdk-samples/Layers/Identify raster cell/IdentifyRasterCell.storyboard

```diff
@@ -0,0 +1,69 @@
+                            <constraint firstItem="cIA-Y7-Zgc" firstAttribute="top" secondItem="84r-Oy-4W5" secondAttribute="top" id="QlH-1u-Gva"/>
+                            <constraint firstItem="cIA-Y7-Zgc" firstAttribute="bottom" secondItem="84r-Oy-4W5" secondAttribute="bottom" id="pr9-bs-jmq"/>
+                        </constraints>
+                        <viewLayoutGuide key="safeArea" id="cQH-id-UEf"/>
+                    </view>
+                    <connections>
+                        <outlet property="calloutStackView" destination="Wqr-jb-4el" id="iS2-bn-zE1"/>
+                        <outlet property="mapView" destination="cIA-Y7-Zgc" id="Hqg-Rn-lha"/>
+                    </connections>
+                </viewController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="P5h-DC-Dtg" userLabel="First Responder" customClass="UIResponder" sceneMemberID="firstResponder"/>
+                <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" distribution="fillProportionally" spacing="10" translatesAutoresizingMaskIntoConstraints="NO" id="Wqr-jb-4el" customClass="IdentifyRasterCellStackView" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target">
+                    <rect key="frame" x="0.0" y="0.0" width="120" height="110"/>
+                    <subviews>
+                        <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Label" textAlignment="natural" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="RaL-Yh-iw7">
+                            <rect key="frame" x="0.0" y="0.0" width="120" height="41"/>
+                            <fontDescription key="fontDescription" type="system" pointSize="12"/>
+                            <nil key="textColor"/>
+                            <nil key="highlightedColor"/>
+                        </label>
+                        <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Label" textAlignment="natural" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="UA8-ez-M5c">
+                            <rect key="frame" x="0.0" y="51" width="120" height="59"/>
+                            <fontDescription key="fontDescription" type="system" pointSize="12"/>
+                            <nil key="textColor"/>
+                            <nil key="highlightedColor"/>
+                        </label>
+                    </subviews>
+                    <constraints>
+                        <constraint firstItem="UA8-ez-M5c" firstAttribute="top" secondItem="RaL-Yh-iw7" secondAttribute="bottom" constant="10" id="AhA-KH-PKH"/>
```

Comment:
Instead of adding a constraint, could you just modify the spacing property of the stack view?

![image](https://user-images.githubusercontent.com/2257493/83200950-4c8f9300-a0f9-11ea-8dd6-de3e7e951d10.png)

Reply:
- [x] Learned a new thing 😄